Dockerfile

```diff
@@ -1,16 +1,11 @@
-FROM golang:1.8.3-alpine
-
+FROM golang:1.8.3-alpine3.6 as unsee-builder
 COPY . /go/src/github.com/cloudflare/unsee
 
 ARG VERSION
+RUN apk add --update make git nodejs nodejs-npm
+RUN make -C /go/src/github.com/cloudflare/unsee VERSION="${VERSION:-dev}" unsee-static
 
-RUN apk add --update --no-cache --virtual .build-dependencies \
-        make git nodejs && \
-    make -C /go/src/github.com/cloudflare/unsee VERSION="${VERSION:-dev}" && \
-    mv /go/src/github.com/cloudflare/unsee/unsee /bin/unsee && \
-    rm -fr /go/src && \
-    apk del .build-dependencies
-
+FROM gcr.io/distroless/base
```

Comment:
What's the advantage of this over using the 'scratch' image and adding ca-certificates as necessary e.g. https://blog.codeship.com/building-minimal-docker-containers-for-go-applications/? It would be good to explain the rationale in your commit message.